terraform/gcp/featureform/terraform.auto.tfvars

```diff
@@ -0,0 +1,12 @@
+# Featureform
+## Cert Manager 
+cert_manager_version = "v1.8.0"
+
+## Featureform Core
+featureform_version     = "v0.1.0"
+featureform_hostname    = ""
```

Comment:
Should we leave this blank so itll error when they try to run it if its not filled?

Reply:
Should I remove it here and the default value as well?

Reply:
Yeah, just force them to do it